action/protocol/context.go

```diff
@@ -127,6 +127,7 @@ type (
 		MigrateNativeStake                      bool
 		AddClaimRewardAddress                   bool
 		EnforceLegacyEndorsement                bool
+		PurgeActpoolOnRunActionError            bool
```

Comment:
thus, we can delete the tag

Reply:
yeah, it does not need to be a hard-fork